Deployment.md

```diff
@@ -25,8 +25,8 @@ metadata:
   labels:
     {{- include "smile.labels" . | nindent 4 }}
 stringData:
-  HOST_IP: "smile-minio"
-  MINIO_URL: "URL for the minio"
+  MINIO_URL: "smile-minio"
+  MINIO_PUBLIC_ACCESS_URL: "Public access URL for the minio"
```

Comment:
I think if you switch to 
```
MINIO_URL: "Public access URL for the minio"
```
Then leave out the MINIO_PUBLIC_ACCESS_URL then it will work :-)